libraries/AP_Logger/AP_Logger_Block.cpp

```diff
@@ -375,8 +455,8 @@ int16_t AP_Logger_Block::get_log_data(uint16_t log_num, uint16_t page, uint32_t
 }
 
 
-// This function determines the number of whole or partial log files in the AP_Logger
-// Wholly overwritten files are (of course) lost.
+// This function determines the number of whole log files in the AP_Logger
+// partial logs are rejected as without the headers they are relatively useless
```

Comment:
tridge noted that we did attempt to add headers; obviously your log can still be corrupt if you add headers later, so I'm happier with this scheme.

Reply:
Yes me too!